packages/desktop/src/renderer/engine.ts

```diff
@@ -13,17 +13,17 @@ export const engine = new HedronEngine({
   onFrameStart: performanceMonitor.begin,
   onFrameEnd: performanceMonitor.end,
   rendererType: import.meta.env.HEDRON_RENDERER_TYPE ?? 'webgl',
+  clock,
 })
 
 export const engineStore = engine.getStore()
 
-export const useEngineStore = <T>(selector?: (state: EngineStateWithActions) => T) => {
-  return useStore(engineStore, selector!)
-}
```

Comment:
`useEngineStore` is no longer coupled to a specific store and can be imported from `@hedron/ui-core`. The new version of this hook uses React context and needs to have `<EngineStoreProvider>` as an ancestor component (e.g. at the root of the React tree), with the store handed there.